habitat_baselines/rl/ppo/ppo_trainer.py

```diff
@@ -578,7 +582,14 @@ def _eval_checkpoint(
 
                 prev_actions.copy_(actions)  # type: ignore
 
-            outputs = self.envs.step([a[0].item() for a in actions])
+            # NB: Move actions to CPU.  If CUDA tensors are
+            # sent in to env.step(), that will create CUDA contexts
+            # in the subprocesses.
+            # For backwards compatibility, we also call .item() to convert to
+            # and int
```

Comment:
typo: an int

Also, technically according to the typing in hablab, it the action doesn't HAVE to be an int does it? It can be a string or anything that corresponds to proper agent config? 

If we enforce it has to be an int, we should adopt the typing in habitat-sim

Reply:
It doesn't have to be an int anymore.  That is just for backwards compatibility (it did once upon a time)

Reply:
Maybe update the comment to reflect that? I think our typing for Simulator in Habitat.core might also need to be updated. :)